simple-counter/src/lib.rs

```diff
@@ -1 +1,86 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::{env, near_bindgen, AccountId};
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct State {
+    count: u64, // TODO,
+}
+
+#[near_bindgen]
+impl State {
+    #[init]
+    pub fn new(count: u64) -> Self {
+        //TODO
+        State { count }
+    }
+
+    pub fn get_num(&self) -> u64 {
+        //TODO
+        self.count
+    }
+
+    pub fn increment(&mut self) {
+        //TODO
+        self.count +=1;
```

Comment:
```suggestion
        self.count += 1;
```

```
cargo +nightly fmt
```

로 포맷팅을 하면 일관된 스타일을 가져갈 수 있을것 같습니다.

Reply:
넵~ 감사합니다